agent/agents/mysql/perfschema/perfschema_test.go

```diff
@@ -328,7 +328,7 @@ func TestPerfSchema(t *testing.T) {
 		}
 
 	default:
-		t.Log("Unhandled version, assuming dummy digests.")
+		t.Logf("Unhandled version, assuming dummy digests. MySQL version: %s, vendor: %s", mySQLVersion, mySQLVendor)
```

Comment:
There is already log with vendor and version little bit above:
https://github.com/percona/pmm/pull/3271/files#diff-0b9b5d27ab40ea00095b446c95db2a4a7ef7c76d651c7f741b4e47789a0c6078R262